bash_completion

```diff
@@ -19,6 +19,8 @@
 #   along with this program; if not, write to the Free Software Foundation,
 #   Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
+#   SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
I did not bother adding this to every source file, I suppose it's good enough this way. But no objections if someone wants to add it to more files, it's just an oneliner anyway.

Reply:
If it is added to all files, I think the copyright notices should also be added to all files at least.

https://github.com/scop/bash-completion/blob/3ddb4452dcff00d48456c6edde1d649440e0e495/bash_completion#L5-L6